components/butterfly/src/message/mod.rs

```diff
@@ -108,7 +109,7 @@ impl Serialize for swim::Rumor {
 
 impl Serialize for swim::Service {
     fn serialize<S>(&self, serializer: &mut S) -> result::Result<(), S::Error> where S: Serializer {
-        let mut state = try!(serializer.serialize_struct("service", 9));
+        let mut state = try!(serializer.serialize_struct("service", 10));
```

Comment:
I wouldn't make any of these constants. They're only used to determine the amount of elements we wish to serialize and the lib only requires them to know how much to allocate